### DISCUSSION TITLE
Integration Support for Salesforce Case Objects

### DISCUSSION THREAD

**Discussion by susifrdmsk** [solspace/craft-freeform#1417] [Feature Requests]
Hello,

I'm using your Salesforce Lead integration on the Freeform plugin, and it's been a big help. But now, we really need to be able to integrate with Salesforce Case objects too. Adding Case object support would let us handle customer support stuff right from Craft CMS, which we need to do ASAP.

Could you possibly look into adding this feature soon? It would make a huge difference for us.
Thanks a lot! 

Kind regards,
Susanne Haberpointner

**Comment by kjmartens** [comment]
Sorry for the delay @susifrdmsk,

This is under consideration to be reviewed again in the future.

If you need this feature right away, we do offer custom software development services where we could expedite building this for you. 🙂 
https://docs.solspace.com/support/premium/
